### ISSUE TITLE
Convnet on the MNIST dataset example has too optimistic test accuracy in comments

### ISSUE THREAD

**Issue by lesshaste** [fchollet/keras#4084]
Please make sure that the boxes below are checked before you submit your issue. Thank you!
- [ Tick] Check that you are up-to-date with the master branch of Keras. You can update with:
  pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps
- [Tick ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
  pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps
- [Tick ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

To get myself started with keras I ran  https://github.com/fchollet/keras/blob/master/examples/mnist_cnn.py . This says in the comments:

```
'''Trains a simple convnet on the MNIST dataset.
Gets to 99.25% test accuracy after 12 epochs
(there is still a lot of margin for parameter tuning).
16 seconds per epoch on a GRID K520 GPU.
'''
```

However, using tensorflow 0.11.0rc0 or the latest theano (0.9.0dev3) I get a test accuracy below 99%. This is on a CPU only (no GPU) set up.  Here is the output of one run using keras and tensorflow.

```
Test score: 0.0315927170954
Test accuracy: 0.9899
```

I have rerun it a few times in case it was random variation causing the problem and I never get above 99%.

It seems the claim of 99.25% accuracy is a little optimistic.
